tests/test_Pretrain.py

```diff
@@ -85,20 +37,3 @@ def test_config1():
   assert_in("hidden_0", net2_json)
   assert_in("hidden_1", net2_json)
   assert_equal(net2_json, net3_json)
-
-
-@unittest.skipIf(not theano, "not theano")
```

Comment:
Does this test actually really depends on Theano? Why?

Reply:
Because it does very explicitly. (`use_theano: True`), but sure, I can re-write it instead.

Reply:
Yea obviously just remove that line (`use_theano`). My question is, does it really depend on Theano? If not, it means nothing needs to be rewritten.